GeosCore/hco_interface_gc_mod.F90

```diff
@@ -4816,7 +4820,6 @@ SUBROUTINE Compute_Sflx_for_Vdiff( Input_Opt,  State_Chm, State_Diag,      &
     RC      =  GC_SUCCESS
     dflx    =  0.0_fp
     eflx    =  0.0_fp
-    spc     => State_Chm%Species(:,:,1,1:State_Chm%nAdvect)
```

Comment:
The pointer spc here is not in the new code.

Reply:
Right, I removed the spc pointer since State_Chm%Species is used directly in Compute_Sflx_for_Vdiff.